packages/clay-management-toolbar/src/SearchForm.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import React from 'react';
+
+interface IProps extends React.FormHTMLAttributes<HTMLFormElement> {
+	onlySearch?: boolean;
+}
+
+const SearchForm: React.FunctionComponent<IProps> = ({
+	children,
+	method = 'get',
+	onlySearch = false,
+	...otherProps
+}) => {
+	const content = (
+		<form {...otherProps} method={method} role="search">
+			{children}
+		</form>
+	);
+
+	if (onlySearch) {
+		return content;
+	}
+
+	return (
```

Comment:
Might make sense to condense to `onlySearch ? content : <div className="container-fluid container-fluid-max-xl">{content}</div>`

Reply:
Yes, that seems clearer. Thanks!